src/mscorlib/corefx/System/Globalization/Calendar.cs

```diff
@@ -840,7 +840,6 @@ internal static long TimeToTicks(int hour, int minute, int second, int milliseco
             throw new ArgumentOutOfRangeException(null, SR.ArgumentOutOfRange_BadHourMinuteSecond);
         }
 
-        [System.Security.SecuritySafeCritical]  // auto-generated
```

Comment:
These attributes will exist as types but the application of the attributes aren't honored and in most cases stale so they don't add any value.